res/database/Default/user/scr0llbaer_constr_films.xml

```diff
@@ -0,0 +1,13364 @@
+				<member index="2" function="2">7902d680-5485-42dd-add8-60b064328be1</member>
+				<member index="3" function="2">3c8502a0-3947-4973-9560-e978fec5af3f</member>
+				<member index="4" function="2">e5123c56-d65e-413d-8517-3c10bf0d46f8</member>
+				<member index="5" function="2">7491c589-c117-42b5-a15d-763ebbc23517</member>
+				<member index="6" function="2">5322bb8e-8061-4c5d-94d6-f585ce99bd8c</member>
+				<member index="7" function="2">ab3f768d-2032-4d3a-b080-edef36d9a881</member>
+				<member index="8" function="2">74b4e934-6d6d-4fea-835c-fead475ea9b1</member>
+				<member index="9" function="2">ec32ea46-be4c-4ef3-b721-77a8fedb9cf3</member>
+				<member index="10" function="2">685277f9-484b-47b2-8495-6a22b267e486</member>
+			</staff>
+			<groups target_groups="0" pro_pressure_groups="0" />
+			<data country="USA" distribution="1" maingenre="5" subgenre="7,18" flags="0" blocks="2" price_mod="1" />
+			<releaseTime year="1993" day="9" />
+			<ratings critics="20" speed="30" outcome="16" />
+		</programme>
+		<programme id="6011ddf4-dffd-45fe-b49e-00f18a7e1e1b" product="1" licence_type="1" creator="9551" imdb_id="tt0107438">
+			<title>
+				<de>Schau mal, wer hier jetzt redet</de>
+				<en>See Who's Speaking Now</en>
+			</title>
+			<title_original>
+				<de>Kuck mal, wer da jetzt spricht</de>
+				<en>Look Who's Talking Now</en>
+			</title_original>
+			<description>
+				<de>Teil 3 der Reihe, der eine haarige Wendung nimmt. Jetzt sind die Haustiere an der Reihe zu quasseln. Rocks ist chaotisch und untrainiert, Daphne hingegen gut erzogen und verwöhnt, und die beiden kommen nicht miteinander klar. Kritiker kläfften: durchwachsen.</de>
+				<en>Part 3 of the series where a furry twist is added. Now it's the pets' turn to gab! Rocks is messy and untrained while Daphne is well-trained and spoiled, and they do not get along. Critics barked mixed reviews.</en>
+			</description>
+			<staff>
+				<member index="0" function="5">cee5b8cc-b2a7-4ecf-833a-b6b42094581f</member>
+				<member index="1" function="2">0a3b1917-3e7e-4fb4-b7c9-8e2ff1ad6b61</member>
+				<member index="2" function="2">DF-Celebreties-Kirstie_Alley</member>
+				<member index="3" function="2">9783239d-01b9-47a5-b053-4874be4f0c70</member>
+				<member index="4" function="2">b0f3e3a1-6f77-43aa-8968-641fcbb9ebaa</member>
+				<member index="5" function="2">c977ab26-224a-47cd-a610-164a05eec977</member>
+			</staff>
+			<groups target_groups="0" pro_pressure_groups="0" />
+			<data country="USA/CDN" distribution="1" maingenre="9" subgenre="5,15" flags="0" blocks="2" price_mod="1" />
+			<releaseTime year="1993" day="11" />
+			<ratings critics="10" speed="32" outcome="23" />
+		</programme>
+		<programme id="d0df86bf-de69-4191-a669-e93279c0dd99" product="1" licence_type="1" creator="9551" imdb_id="tt0107076">
+			<title>
+				<de>Heftige Ziele</de>
+				<en>Heavy Target</en>
+			</title>
+			<title_original>
+				<de>Harte Ziele</de>
+				<en>Hard Target</en>
+			</title_or
```

Comment:
duplicate name is intentional!?

Reply:
Which? Display here seems buggy, seems you refer to "Dr. Dumlitter/Doolittle"? Yes, the second one is a remake.